app/javascript/src/NewService/components/FormElements/ErrorMessage.jsx

```diff
@@ -0,0 +1,14 @@
+// @flow
+
+import React from 'react'
+
+const ErrorMessage = ({error}: {error: string}) => {
+  const classNames = `errorMessage ${error ? '' : 'is-hidden'}`
+  return <p className={classNames}>
+    {`Sorry, your request has failed with the error: ${error}`}
```

Comment:
The error comes from the request response, so yes. 
What error do you think is more to the user @thomasmaas ?
We have something like this:
`Sorry, your request has failed with the error: Cannot read property 'projects' of undefined`

Reply:
>The error comes from the request response, so yes.
Ok, in that case it could be renamed `FetchErrorMessage` or `NetworkErrorMessage`

Reply:
@damianpm in order to write something in 'normal' english i first need to know what the error represents.

Reply:
@thomasmaas  It will be a fetch error: 
`Cannot read property 'projects' of undefined`
or
`Cannot read property 'services' of undefined`
Additionally, the browser shows a full error message like:
<img width="760" alt="error-message" src="https://user-images.githubusercontent.com/13486237/58091162-32a7a000-7bc9-11e9-9044-d0ee27405cfb.png">